### PR TITLE
b2bevents mixins

### PR DESCRIPTION
**Pull request by prabhum2**
Please link to the issue #…

### CODE REVIEW

File: components/mixins/experience-event/events/visitwebpage.schema.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        "xdm:web": {
+          "title": "Web",
+          "type": "object",
+          "properties": {
+            "xdm:webPageDetails": {
+              "title": "Web Page Details",
+              "type": "object",
+              "properties": {
+                "xdm:name": {
+                  "title": "Name",
```
Comment (peichiang) [inline components/mixins/experience-event/events/visitwebpage.schema.json:30]: Assume this is "Webpage ID" from Marketo side.
Let's capture this mapping in our wiki to help facilitate the discussion - whether to keep id and/or name for an object reference.

**Review by peichiang — APPROVED** [review]
Looks good to me for now.

**Review by mjasrotia — APPROVED** [review]

File: components/mixins/experience-event/events/removefromlist.schema.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/removefromlist",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Remove From List",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "removefromlist",
```
Comment (cmathis) [inline components/mixins/experience-event/events/removefromlist.schema.json:13]: This conditional value is what they pass to the "eventType" field. Please look at the existing list of potential values and use it, or propose a new value here. Use the same format if creating a new value - https://github.com/adobe/xdm/blob/master/components/behaviors/time-series.schema.json#L

File: components/mixins/experience-event/events/visitwebpage.schema.json [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/visitwebpage",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Visit WebPage",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "visitwebpage",
```
Comment (cmathis) [inline components/mixins/experience-event/events/visitwebpage.schema.json:13]: This should be web.webpagedetails.pageViews
https://github.com/adobe/xdm/blob/master/components/behaviors/time-series.schema.json#L45

**Review by cmathis — APPROVED** [review]

**Pull request merged**